cmd/manager_logging.go

```diff
@@ -174,6 +174,18 @@ var (
 						Action: runAddSMTPLogger,
 					},
 				},
+			}, {
+				Name:  "log-sql",
+				Usage: "Set LogSQL",
+				Flags: []cli.Flag{
+					cli.BoolFlag{
+						Name: "debug",
+					}, cli.BoolFlag{
+						Name:  "off",
```

Comment:
Honestly I don't know what to do here.

@wxiaoguang thought having separate named methods wasn't good.

I think it's perfectly reasonable to default to on. I can add a `--on` flag but it may as well do nothing as the default will always be `on`

Reply:
ah but the boolean is showSQL down in the xorm layer.

Reply:
Yeah, I'm fine with both approaches, simply wanted to point out that making it uniform might also be beneficial.